knnvector/README.md

```diff
@@ -0,0 +1,169 @@
+# KNN Vector Workload
+
+This workload is to benchmark performance of indexing and search using knn_vector as field type.
```

Comment:
Lets change this to something, This workload is to benchmark performance of indexing and search of Vector Engine of Opensearch

Reply:
Do you think we should change workload name from knn vector to Vector engine too?